hintrc.txt

```diff
@@ -0,0 +1,18 @@
+{
+  "connector": {
+    "name": "local",
+    "options": {
+      "pattern": ["**", "!.git/**", "!node_modules/**"]
+    }
+  },
+  "extends": ["development"],
+  "formatters": ["stylish"],
+  "hints": [
+    "button-type",
+    "disown-opener",
+    "html-checker",
+    "meta-charset-utf-8",
+    "meta-viewport",
+    "no-inline-styles:error"
+  ]
+}
```

Comment:
- [ ] Excellent work on your first project! I have a **few suggestions that could help make it even better.** Could you please take a moment to check the name of this file? It should be named as `.hintrc` instead of `hintrc.txt`. Making this small adjustment **will ensure everything is perfect.** 😄

- [ ] Could you kindly include the requested **styleint** configuration file linters? You can find the file in the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), which are available here: [Link](https://github.com/microverseinc/linters-config/tree/master/html-css). Just a reminder, the name of the styleint configuration file should be `.stylelintrc.json`. 👌

- [ ] I kindly request that you refrain from modifying the name of the linters file inside the workflows folder. Currently, it's named `linters (1).yml`, but the correct name should simply be `linters.yml`. Your attention to this detail would be greatly appreciated. 💪

   ![image](https://github.com/enocol/HelloMicroverse/assets/107952724/769d6eef-3814-454c-af8f-ef6b41a43463)

Reply:
Thanks for pointing that out. The files were not renamed. The reason is that I had several copies of this file in my download folder so the systems named them systematically. It has been corrected now, thanks